java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -238,37 +217,34 @@ protected Void doInBackground(Void... arg0) {
 		}
 		
 		@Override
-        protected void onPostExecute(Void result) {
-            progressDialog.dismiss();
+		protected void onPostExecute(Void result) {
+			mProgressDialog.dismiss();
         }
 	}
 	
 	public class MainActivityCallback<Object> implements RequestCallback {
-    	public final static int UpdateRecords = 0;
-    
-    	private int mEvent;
-    	
-    	public MainActivityCallback(int event) {
-    		MainActivity.this.setProgressBarIndeterminateVisibility(true);
+		public final static int UpdateRecords = 0;
+
+		private int mEvent;
+
+		public MainActivityCallback(int event) {
+			MainActivity.this.setProgressBarIndeterminateVisibility(true);
 			mEvent = event;
-    	}
+		}
 
-        @Override
-        public void onError(HVException exception) {
-        	MainActivity.this.setProgressBarIndeterminateVisibility(false);
-            Toast.makeText(
-    		MainActivity.this, 
-                "An error occurred.  " + exception.getMessage(), 
-                Toast.LENGTH_LONG).show();
-        }
+		@Override
+		public void onError(HVException exception) {
+			MainActivity.this.setProgressBarIndeterminateVisibility(false);
+			Toast.makeText(MainActivity.this, "An error occurred.  " + exception.getMessage(), Toast.LENGTH_LONG).show();
+		}
 
 		@Override
 		public void onSuccess(java.lang.Object obj) {
 			MainActivity.this.setProgressBarIndeterminateVisibility(false);
-            switch(mEvent) {
-            case UpdateRecords:
-                break;
-            }
-        }
-    }
+			switch(mEvent) {
```

Comment:
Is this being updated to add something? If so add a TODO:, otherwise we can delete empty switch